README.md

```diff
@@ -85,6 +79,91 @@ Given that we have a DRO with two Filesets each with a File (image1.png) and (im
 1. POST /filesets with the `signed_id` from step one.  Repeat for the second file. The API will use `ActiveStorage::Blob.find_signed(params[:signed_id])` to find the files.
 1. POST /dro with the fileset ids from the previous step.
 
+## User management
+### Create a user
+```
+bin/rake "users:create[leland@stanford.edu,sekret]"
+#<User:0x000000010df4c4d8> {
+                 :id => 1,
+               :name => nil,
+              :email => "leland@stanford.edu",
+   "password_digest" => "[DIGEST]",
+         :created_at => Thu, 17 Nov 2022 17:26:28.927725000 UTC +00:00,
+         :updated_at => Thu, 17 Nov 2022 17:26:28.927725000 UTC +00:00,
+             :active => true,
+        :full_access => true,
+        :collections => []
+}
```

Comment:
A minor tweak: what would you think about prompting for the password value so we don't have them stored in shell history? Something like https://github.com/sul-dlss/infrastructure-integration-test/blob/main/spec/support/authentication_helpers.rb#L55-L59

Reply:
Done.